web/models/user.ex

```diff
@@ -78,11 +81,22 @@ defmodule CodeCorps.User do
   def update_changeset(struct, params) do
     struct
     |> changeset(params)
-    |> cast(params, [:first_name, :last_name, :twitter, :biography, :website, :base64_photo_data])
+    |> cast(params, [:first_name, :last_name, :twitter, :biography, :website, :base64_photo_data, :state_transition])
```

Comment:
Think maybe we should `validate_inclusion` on the `state_transition` so the developer can track down the error. Even though `apply_state_transition()` does add one, that might be worthwhile to do.